acceptance/bundle/variables/bundle_git_branch_issue3291/script

```diff
@@ -0,0 +1 @@
+databricks bundle deploy --var branch=my_branch
```

Comment:
We need to run `bundle validate -o json` to assert on the reported issue. This does not check the variable value.